tests/app/dao/test_api_key_dao.py

```diff
@@ -95,3 +95,16 @@ def test_save_api_key_should_not_create_new_service_history(sample_service):
     save_model_api_key(api_key)
 
     assert Service.get_history_model().query.count() == 1
+
+
+def test_should_not_return_revoked_api_keys_older_than_7_days(sample_service):
+    expired_api_key = ApiKey(**{'service': sample_service,
+                                'name': sample_service.name,
+                                'created_by': sample_service.created_by,
+                                'key_type': KEY_TYPE_NORMAL,
+                                'expiry_date': datetime.utcnow() - timedelta(days=8)})
+    save_model_api_key(expired_api_key)
+
+    all_api_keys = get_model_api_keys(service_id=sample_service.id)
+
+    assert len(all_api_keys) == 0
```

Comment:
can you add a second key with expiry date of 6 days ago and make sure that's in the returned list?